client/src/app/UI/side-panel/tabs/roi/roiitem/roiitem.component.html

```diff
@@ -73,11 +73,11 @@
             </two-state-icon-button>
             <div fxLayout="column" class="gap-separated-vertical-elements" class="label-container">
                 <div class="side-panel-label" title="{{labelToShow}}">{{labelToShow}}</div>
-                <div *ngIf="level !== null || dateAdded !== null" class="metadata-line" title="Level">
+                <div *ngIf="level !== null || dateAdded > 0" class="metadata-line" title="Level">
```

Comment:
Ah I see this is the check you were talking about - this might cause some issues with already uploaded MIST ROIs (won't show the dots representing level) as it looks like in javascript any non-numeric string will return false with the `>` operator. This probably should be corrected for in the API anyways, but just need to make sure the API changes are deployed at the same time as this then!